packages/xo-vmdk-to-vhd/src/vmdk-to-vhd.integ.spec.js

```diff
@@ -71,7 +71,7 @@ test('VMDK to VHD can convert a random data file with VMDKDirectParser', async (
       )
     ).pipe(createWriteStream(vhdFileName))
     await fromEvent(pipe, 'finish')
-    await execa('vhd-util', ['check', '-p', '-b', '-t', '-n', vhdFileName])
+    await execa('qemu-img', ['convert', '-fvpc', '-Oqcow2', vhdFileName, 'outputFile.qcow2'])
```

Comment:
Comment this line and/or extract it in a function to make it easier to understand.

Reply:
Can I use the ```checkFile()``` function from ```tests/utils.js```? It does exactly the same and it would be more readable.

Reply:
Unfortunately no, it's not in the same package, you can copy/paste it though.